abdulfatai360/banka
# #165805750 Implement user registration feature

#### What does this PR do?
Implement user registration feature on frontend

#### Description of Task to be completed?
- Have inputs on signup form validated on client side
- Have the signup API integrated with the signup form
- Have user notified when registration is not successful
- Have user redirected to the user dashboard on successful registration
- Have the user profile section on the dashboard populated with the user information

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/60/head:ft-register-user-165805750`
- Now that you have a copy of the branch, switch to it using  `git checkout ft-register-user-165805750`
- Your directory will now be an exact copy of the PR

#### What are the relevant pivotal tracker stories?
[#165805750](https://www.pivotaltracker.com/story/show/165805750)
